### PR TITLE
feat(misc) - remove removeFindDomNode prop

### LINKED ISSUE #161: Various - removeFindDomNode prop was removed
Follow up to breaking change PR #8316 as well as #8371

`removeFindDomNode` was removed and has been made the default implementation. This might be more of an epic but the codemods I would think would be very similar to each other if it couldn't be done in a single codemod.

Affected components which need a codemod to remove the prop:
- [ ] ApplicationLauncher
- [ ] ClipboardCopy
- [ ] ContextSelector
- [ ] Dropdown
- [ ] NavItem
- [ ] OptionsMenu
- [ ] Popover
- [ ] SearchInput
- [ ] Select
- [ ] OverflowTab
- [ ] Timepicker
- [ ] Tooltip
- [ ] Truncate

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by kmcfaul**
Closes #161

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+const rule = require('../../../lib/rules/v5/remove-removeFindDomNode');
+
+ruleTester.run("remove-removeFindDomNode", rule, {
+  valid: [
+    {
+      code: `import { ApplicationLauncher } from '@patternfly/react-core'; <ApplicationLauncher />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<ApplicationLauncher />`,
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js:11]: The "no pf import" tests actually are to test that non-pf components are not impacted, so the property should be in there.

**Review by gitdallas — CHANGES_REQUESTED** [review]

File: packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js (hunk 1 above)
Comment (gitdallas) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js:11]: Also, should the readme be updated?

Comment (kmcfaul) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js:11]: Yep, I figured that distinction out a bit later as I was testing another codemod haha. I'll update the test and also the readme.

**Review by gitdallas — APPROVED** [review]
i'd have mapped out the renames and rules just so it's easier to see what's all there and reduce chance of missing something, but otherwise lgtm

File: packages/pf-codemods/README.md [hunk 2]
```diff
@@ -40,6 +40,48 @@ Options:
 
 These rules are based off the breaking change notes for React. Each rule links the breaking change patternfly-react PR in case you want to better understand the change. Also, each rule makes sure you're using a PatternFly component before running.
 
+### remove-removeFindDomNode [(#8371)](https://github.com/patternfly/patternfly-react/pull/8371) [(#8316)](https://github.com/patternfly/patternfly-react/pull/8316)
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:43]: Could be worth placing alphabetically after resolving the conflicts

**Review by thatblindgeye — COMMENTED** [review]
Looks good! Had a comment below, but otherwise just needs conflicts resolved

**Pull request merged**